cmd/mayactl/app/command/volume_test.go

```diff
@@ -690,3 +690,69 @@ func TestGetNodeName(t *testing.T) {
 		})
 	}
 }
+
+func TestGetVolumeNamespace(t *testing.T) {
+	tests := map[string]struct {
+		expectedOutput string
+		Volume         VolumeInfo
+	}{
+		"Fetching volume namespace when present in response": {
+			Volume: VolumeInfo{
```

Comment:
The logic of creating a VolumeInfo is duplicated in this UT.
Can you create a private function to remove this duplication?
e.g. `func mockVolumeInfoFromNS(ns string) VolumeInfo`